saltstack/base/salt/top.sls

```diff
@@ -15,4 +15,5 @@ base:
     - fluent
 {% endif %}
     - ccm-client
+    - cis-controls
```

Comment:
@bormanbhaskar it might make sense to make this conditional based on an environment variable. I think this would make it easier to opt in/out during the build process. The fluent example is probably good to follow.

The rationale here is that it is much easier to change an environment variable - than it is to change the code if we need to revert this for any reason during testing.

Reply:
Added conditional statement inside init.sls